src/components/landing/ScrollAnimations.tsx

```
@@ -0,0 +1,73 @@
+      (entries) => {
+        for (const entry of entries) {
+          if (!entry.isIntersecting) continue;
+          (entry.target as HTMLElement).setAttribute("data-visible", "");
+        }
+      },
+      { threshold: 0.1, rootMargin: "0px 0px -100px 0px" },
+    );
+
+    root
+      .querySelectorAll<HTMLElement>("[data-fade-in]")
+      .forEach((el) => fadeObserver.observe(el));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)